src/editorialPalette.ts

```diff
@@ -80,32 +83,20 @@ const border = (format: Format): Colour => {
     }
 }
 
-const fontColour = (colour: Colour): SerializedStyles =>
-    css`
-        color: ${colour.light};
-
-        @media (prefers-color-scheme: dark) {
-            color: ${colour.dark};
-        }
-    `;
-
-const backgroundColour = (colour: Colour): SerializedStyles =>
-    css`
-        background-color: ${colour.light};
-
-        @media (prefers-color-scheme: dark) {
-            background-color: ${colour.dark};
-        }
-    `;
-
-const headlineFontColour = compose(fontColour, headlineFont);
-const headlineBackgroundColour = compose(backgroundColour, headlineBackground);
+const palette = (format: Format): Palette =>
+    ({
+        headlineFont: headlineFont(format),
+        headlineBackground: headlineBackground(format),
+        border: border(format),
+    });
```

Comment:
_This_. It's funny how it looks so simple when you see it here now but I guess the solutions that seem simple are the best ones!

One nitpick, I wouldn't name the function the same as the property it is setting, I normally go for some kind of prefix like `decideHeadlineBackground()` or `getHeadlineBackground()` but that's maybe because I lean a lot on using `CMD+D` to select similar text when editing. Defo not a blocker.

Reply:
I suspect DCR might make use of this (with a theme), and AR might make use of the individual functions/CSS declarations.